delta/sourcing-psql/src/test/scala/ch/epfl/bluebrain/nexus/delta/sourcing/EntityDependencyStoreSuite.scala

```diff
@@ -58,7 +59,9 @@ class EntityDependencyStoreSuite extends BioSuite with Doobie.Fixture {
   private val dependencyId5 = DependsOn(proj2, id5)
 
   test("Save the different states") {
-    List(state1, state2, state3, state5).traverse(stateStore.unsafeSave(_)).transact(xas.write)
+    val saveLatestStates = List(state1, state2, state3, state5).traverse(stateStore.unsafeSave(_))
+    val saveTaggedStates = List(state2, state3).traverse(stateStore.unsafeSave(_, UserTag.unsafe("my-tag")))
+    (saveLatestStates >> saveTaggedStates).transact(xas.write)
```

Comment:
I can't really tell what this test is doing, is this affecting the other tests somehow?

Reply:
It is populating the database with tagged states that should never been returned in this context

Reply:
The problem I see is that it's unclear from reading the test how those tagged states would affect anything. It would be better if there was a test that was clearly saying that the tagged versions should not appear (or whatever the assertion should be)